### DISCUSSION TITLE
Need Guidance! npm run dev:build

### DISCUSSION THREAD

**Discussion by omairakash1** [withcatai/node-llama-cpp#310] [Q&A] [answered]
when I do npm run dev:build, it gets stuck at this point, I checked task manager processes it seems to be running ld.exe

![image](https://github.com/user-attachments/assets/aa98c9d6-792f-41d9-aaa3-6f519d3e39f6)

**Comment by omairakash1** [comment]
![image](https://github.com/user-attachments/assets/cc91095f-5722-485d-bd39-53ce2113d7ad)

**Reply by giladgd** [reply, marked as answer]
There was recently a breaking change in `llama.cpp` and the fix isn't ready yet.
I'm actively working on it and will release it soon.

**Reply by omairakash1** [reply]
thanks
